### PR TITLE
[Release] Bump version to 1.0.0-alpha.9

### PR DESCRIPTION
**Pull request by feltech**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -10,31 +10,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # Constrain to single version to simplify install from wheels
-        os: ['ubuntu-latest']
-        python: ['3.7']
-        # os: ['windows-latest', 'ubuntu-latest', 'macos-latest']
-        # python: ['3.7', '3.9', '3.10']
+         os: ['windows-latest', 'ubuntu-latest', 'macos-latest']
+         python: ['3.7', '3.9', '3.10']
```
Comment (elliotcmorris) [inline .github/workflows/test.yml:14]: [Consider] We might want to add 3.11 to this now as well.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
-v1.0.0-alpha.x
+v1.0.0-alpha.9
 --------------
 
 ### Breaking Changes
 
 - Minimum OpenAssetIO version increased to `v1.0.0-alpha.13` due to API
-  changes.
+  changes. In particular, `getRelatedReferences` has changed to
+  `getWithRelationship` and `getWithRelationships`.
```
Comment (elliotcmorris) [inline RELEASE_NOTES.md:11]: [Fix] I don't personally think it's appropriate to make specific mention of openassetio methods in the BAL release notes. This isn't a consideration that BAL consumers need to care about, in theory. What do you think?

**Review by elliotcmorris — CHANGES_REQUESTED** [review]

File: RELEASE_NOTES.md (hunk 2 above)
Comment (feltech) [reply RELEASE_NOTES.md:11]: My thought was slightly selfish, in that I've found myself looking at release notes trying to remember whether the `getWithRelationship` rename was already released or not, but it wasn't mentioned at all in the release notes.

File: .github/workflows/test.yml (hunk 1 above)
Comment (feltech) [reply .github/workflows/test.yml]: Added in 9fb9a555

File: RELEASE_NOTES.md (hunk 2 above)
Comment (feltech) [reply RELEASE_NOTES.md]: Dropped the commit that updated the release notes.

Comment (elliotcmorris) [reply RELEASE_NOTES.md]: Thanks for that, sorry for being a pain.

**Review by elliotcmorris — APPROVED** [review]

**Pull request merged**